src/Platforms/AbstractPlatform.php

```diff
@@ -3020,6 +3024,16 @@ public function getCurrentTimestampSQL()
         return 'CURRENT_TIMESTAMP';
     }
 
+    /**
+     * Returns the SQL specific for the platform to get an empty JSON object
+     *
+     * @return string
+     */
+    public function getJsonObjectSQL()
+    {
+        return 'JSON_OBJECT';
```

Comment:
Are you sure this is portable across all supported platforms that support JSON columns?